File: RubricaXMLManager/Manager/src/io/ghostyjade/manager/addressbook/AddressBookEntry.java
package io.ghostyjade.manager.addressbook;

public class AddressBookEntry {

	//Required infos
	private String name;
	private String surname;
	private String phoneNumber;

	private String entryMail;
	private String address;
	private String webAddress;
	private String notes;
	private String bornDate;
	private String nickname;

	public AddressBookEntry(String name, String surname, String phoneNumber) {
		this.name = name;
		this.surname = surname;
		this.phoneNumber=phoneNumber;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getSurname() {
		return surname;
	}

	public void setSurname(String surname) {
		this.surname = surname;
	}

	public String getEntryMail() {
		return entryMail;
	}

	public void setEntryMail(String entryMail) {
		this.entryMail = entryMail;
	}

	public String getPhoneNumber() {
		return phoneNumber;
	}

	public void setPhoneNumber(String phoneNumber) {
		this.phoneNumber = phoneNumber;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}

	public String getWebAddress() {
		return webAddress;
	}

	public void setWebAddress(String webAddress) {
		this.webAddress = webAddress;
	}

	public String getNotes() {
		return notes;
	}

	public void setNotes(String notes) {
		this.notes = notes;
	}

	public String getBornDate() {
		return bornDate;
	}

	public void setBornDate(String bornDate) {
		this.bornDate = bornDate;
	}

	public String getNickname() {
		return nickname;
	}

	public void setNickname(String nickname) {
		this.nickname = nickname;
	}

}

File: RubricaXMLViewer/RubricaXMLViewer/RubricaXMLViewer/AddressBook/Data/AddressBookEntry.cs
using System.Collections.Generic;

namespace RubricaXMLViewer.AddressBook.Data
{
    public class AddressBookEntry
    {
        public string Name { get; set; }
        public string Surname { get; set; }
        public string PhoneNumber { get; set; }

        public string EntryMail { get; set; }
        public string Address { get; set; }
        public string WebAddress { get; set; }
        public string Notes { get; set; }
        public string BornDate { get; set; }
        public string Nickname { get; set; }

        public AddressBookEntry(string name, string surname, string pn, string em, string address, string wa, string notes, string bd, string nickname)
        {
            Name = name;
            Surname = surname;
            PhoneNumber = pn;
            EntryMail = em;
            Address = address;
            WebAddress = wa;
            Notes = notes;
            BornDate = bd;
            Nickname = nickname;
        }
        public AddressBookEntry() { }
    }
}
